openhtf/names.py

```diff
@@ -55,5 +56,6 @@
 # Classes used in writing test scripts.
 Measurement = openhtf.util.measurements.Measurement
 OutputToJSON = openhtf.io.output.json_factory.OutputToJSON
+UploadJSON = openhtf.io.output.upload_json.UploadJSON
```

Comment:
In the last meeting, we discussed the 'vision' for names.py and generally agreed it should only include modules that are vendor-agnostic. While the JSON is vendor-agnostic, upload_json is centered around uploading to the 'CM' system, so it doesn't belong in names.py, sorry!